showcase/openfe_showcase.ipynb

```diff
@@ -2,12 +2,11 @@
  "cells": [
```

Comment:
<p>Maybe to be discussed?</p><p> </p><p>Overall this section confuses me a bit because we're doing things in vastly different ways for the two approaches.</p><p> </p><p>Would it make more sense to "just" create the Transformations, and then for the "executing in Python" portion we use "Transformation.create()" to get the DAG out and run that?</p><p> </p><p>Then in the CLI portion we can use the same Transformations, write them to disk and say we can use quickrun to run them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>